custom_components/tuya_local/devices/smartmi_e1_purifier.yaml

```diff
@@ -0,0 +1,61 @@
+name: Smartmi Air Purifier E1
+products:
+  - id: pwrmggdhwkz1lnvh
+    name: Smartmi Air Purifier E1
+primary_entity:
+  entity: fan
+  translation_key: fan_with_presets
+  name: Air Purifier
```

Comment:
This will be appended to the device name, so repeating air purifier here is redundant. No name is OK for the primary entity.